### PR TITLE
updating html (using bootstrap v4.0.0 classes) and css to incorporate mobile responsive page

### PR DESCRIPTION
**Pull request by geraldbcastillo**
## Background
I personally use the site to watch everything in order (thank you by the way) and noticed some funky styles when using my phone. I wanted to contribute to something that I regularly use and hopefully help out!

## Desktop
Moved the title and _nav links_ into a **Bootstrap navbar** to keep the top area clean and make it versatile for a collapsible menu in mobile (see below)
desktop

## Mobile
* Moving navbar functions into a collapsible menu
* Adding responsiveness to filter form
* Adding responsiveness to table
mobile

### CODE REVIEW

**Comment by AceFire6** [comment]
Hi @geraldbcastillo 👋 

Thank you for taking the time to make a pull request. I really appreciate it! I'll try and review this as soon as possible 👍

File: ordering/templates/index.html [hunk 1]
```diff
@@ -7,225 +7,207 @@
     <meta name="description" content="Arrowverse episode order tracker. Never spoil a crossover episode again!">
 
     <title>Arrowverse Episode Order</title>
-    <link href="{{ url_for('recent_episodes', hide_show=hidden_show_list) }}"
-          rel="alternate"
-          title="Recent Episodes"
-          type="application/atom+xml">
+    <link href="{{ url_for('recent_episodes', hide_show=hidden_show_list) }}" rel="alternate" title="Recent Episodes"
```
Comment (AceFire6) [inline ordering/templates/index.html]: This wrapping shouldn't have changed

File: ordering/templates/index.html [hunk 2]
```diff
@@ -7,225 +7,207 @@
...
-                                    <option value="{{ show_id }}" {% if show_id in hidden_show_list %}selected{% endif %}>
-                                        {{ show_data.name }}
-                                    </option>
-                                {% endfor %}
-                            </select>
-                        </div>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+        <div class="container">
+            <span class="navbar-brand">Arrowverse Episode Order</span>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
```
Comment (AceFire6) [inline ordering/templates/index.html]: I'd prefer this to be wrapped so it's easier to see all the attributes on the element

File: ordering/static/css/index.css [hunk 3]
```diff
@@ -102,3 +96,11 @@ a.btn {
...
     background-color: transparent;
 }
+
+nav.navbar {
+    margin-bottom: 1rem;
+}
+
+table.table {
+    margin: 0;
+}
```
Comment (AceFire6) [inline ordering/static/css/index.css]: Newline missing at the end of the file

File: ordering/templates/index.html [hunk 4]
```diff
@@ -7,225 +7,207 @@
...
+                src="https://cdn.jsdelivr.net/npm/daterangepicker/daterangepicker.min.js"></script>
+            <script src="{{ url_for('static', filename='js/index.js') }}"></script>
+            <script src="{{ url_for('static', filename='js/cookie.js') }}"></script>
+            {% include 'analytics.html' %}
+        </div>
+    </main>
 </body>
-</html>
+
+</html>
```
Comment (AceFire6) [inline ordering/templates/index.html]: Newline missing at the end of the file

**Comment by AceFire6** [comment]
@geraldbcastillo If possible can you merge master, please? I've added the option to make review apps with Heroku which should make reviewing the visual changes a lot simpler 👍

File: ordering/static/css/index.css (hunk 3 above)
Comment (geraldbcastillo) [reply ordering/static/css/index.css]: New line added!

**Pull request closed without merging**